channels/packages/node-manager/develop/deployment-storageos-node-manager.yaml

```diff
@@ -38,29 +38,63 @@ spec:
                 app.kubernetes.io/component: node-manager
             topologyKey: kubernetes.io/hostname
       containers:
-      - args:
-        - --config=node_manager_config.yaml
+      - name: manager
```

Comment:
You have to edit manifests at `node-manager` repo, merge change, CI would build develop images, and you have to execute `make node-manager` at this repo.